package/MDAnalysis/topology/guessers.py

```diff
@@ -48,15 +48,30 @@ def guess_masses(atom_types):
     -------
     atom_masses : np.ndarray dtype float64
     """
+    validate_atom_types(atom_types)
     masses = np.array([get_atom_mass(atom_t) for atom_t in atom_types], dtype=np.float64)
-    if np.any(masses == 0.0):
-        # figure out where the misses were and report
-        misses = np.unique(np.asarray(atom_types)[np.where(masses == 0.0)])
-        warnings.warn("Failed to guess the mass for the following atom types: {}"
-                      "".format(', '.join(misses)))
     return masses
 
 
+def validate_atom_types(atom_types):
```

Comment:
The docstring for this is raising an error in the doc build test, I think it might be that the keyword is `Parameters`

Reply:
Fixed the typo. Please review.